src/styles/card.css

```
@@ -0,0 +1,100 @@
+.card {
+  font-family: inherit;
+
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  display: block;
+  height: 134px;
+  width: 160px;
+  margin: 0 auto;
+}
+
+.card__block-description {
+  margin-top: 40px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--primary-color);
+}
+
+.card__block-product {
+  margin-top: 4px;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  text-align: left;
+  color: var(--secondary-color);
+}
+
+.card__block-rating {
+  margin-top: 16px;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__block-reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--primary-color);
+}
+
+.card__block-cost {
+  margin-top: 24px;
+
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__block-price {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary-color);
+}
+
+.card__block-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--primary-color);
+}
+
+.card__block-link {
+  display: block;
+  width: 166px;
+  height: 40px;
+
+  margin-top: 16px;
+  padding: 12px 70px;
+
+  border-radius: 5px;
+  background-color: var(--button-color);
+  border: 1px solid transparent;
+  text-align: center;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  color: #fff;
+
+  transition: all 0.3s;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```

---

**Fixed**